driver-core/src/main/com/mongodb/connection/ServerDescription.java

```diff
@@ -89,6 +90,7 @@
     private final long lastUpdateTimeNanos;
 
     private final Integer logicalSessionTimeoutMinutes;
+    private final TopologyVersion topologyVersion;
```

Comment:
Topology version support shouldn't be added as part of this ticket.  I'm guessing you picked that up from the new tests or spec changes added by Shane?

Reply:
Its in the latest version of the spec

Reply:
Removed